### PR TITLE
Adds Airtable support

### PR DESCRIPTION
**Pull request by der-On**
This PR adds support for airtable.com. As it is almost impossible to guess tags or projects from an airtable dataset the integration only uses the datasets identifier as the description.

toggl-button_airtable

### CODE REVIEW

**Comment by der-On** [comment]
@syko any chance of getting this merged?

**Comment by der-On** [comment]
I've now resolved the conflicts and would really like some feedback on this PR. Are there any things left to be done to get this merged?

File: src/scripts/content/airtable.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*jslint indent: 2 */
+/*global $: false, document: false, togglbutton: false*/
+
+'use strict';
+
+togglbutton.render('.detailViewWithActivityFeedBase .dialog > .header > .flex-auto:not(.toggl)', {observe: true}, function (elem) {
+
+  var link, descFunc,
+    container = $('.justify-center > .relative', elem),
+    description = $('.truncate.flex-auto.line-height-3', elem);
```
Comment (Phenome) [inline src/scripts/content/airtable.js]: Sadly those elements don't have more deterministic class names, so I'd go with perhaps `:nth-child` selectors. But probably not that much more reliable anyway.
These work for current interface layout, so this is good to go :)

**Comment by der-On** [comment]
@Phenome thank you!

**Pull request merged**